rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -148,11 +148,12 @@ protected Class getJsonFormActivityClass() {
     }
 
     @Override
-    protected void prePopulateRDTPatientFields(Patient patient, JSONObject field) throws JSONException {
-        super.prePopulateRDTPatientFields(patient, field);
+    protected void prePopulateRDTPatientFields(Patient patient, JSONObject field, Context context) throws JSONException {
+        super.prePopulateRDTPatientFields(patient, field, context);
         String key = field.getString(JsonFormUtils.KEY);
         if (PATIENT_SEX.equals(key)) {
-            field.put(JsonFormUtils.VALUE, patient.getPatientSex().toLowerCase());
+            String translatedSex = patient.getPatientSex();
+            field.put(JsonFormUtils.VALUE, context.getString(R.string.female).equalsIgnoreCase(translatedSex) ? "female" : "male");
```

Comment:
Could we move `"female" : "male"` to constants?

Reply:
done